app/javascript/article-form/components/__tests__/Close.test.jsx

```diff
@@ -15,6 +15,6 @@ describe('<Close />', () => {
     const { getByTitle } = render(<Close />);
     const icon = getByTitle(/Close the editor/i);
 
-    expect(icon.closest('a').getAttribute('href')).toEqual('/');
+    expect(icon).toBeDefined();
```

Comment:
(nitpick / not blocking) - I feel a bit like `toBeInTheDocument` would be more descriptive for this one
```suggestion
    expect(icon).toBeInTheDocument();
```